ontologies/vocabs.ttl

```diff
@@ -1083,6 +1090,41 @@ The tool was unable to determine whether the malware binary is malicious."""^^xs
 	skos:prefLabel "unknown"^^xsd:string ;
 	.
 
+gist:_NetworkSocketType_SOCK_DGRAM
+	a gist:NetworkSocketType ;
+	skos:definition """STIX 2.1 description:
+Specifies a socket in which individually-addressed packets are sent (datagram)."""^^xsd:string ;
+	skos:prefLabel "SOCK_DGRAM"^^xsd:string ;
+	.
```

Comment:
```suggestion
gist:_NetworkSocketType_datagram
	a gist:NetworkSocketType ;
	skos:definition """STIX 2.1 description:
Specifies a socket in which individually-addressed packets are sent (datagram)."""^^xsd:string ;
skos:prefLabel "Datagram"^^xsd:string ;
skos:altLabel "SOCK_DGRAM"^^xsd:string ; 
```
Labels should generally be human-friendly and echo IRIs.  This is one option, an alternative is to keep same IRI and prefLabel and add a human-friendly label as altLabel. I guess the SOCK_ prefixes have an XML heritage?

NOTE: ignore the indent changes, they are artifacts of github.